app/screens/Settings/__tests__/__snapshots__/SettingsScreen.test.tsx.snap

```diff
@@ -40,6 +40,55 @@ exports[`SettingsScreen renders correctly 1`] = `
         size={14}
       />
     </TouchableOpacity>
+    <View
```

Comment:
Nice to have : if you add the following two lines to `jest/mock/components/index.js` you will mock globally `ListItem` and `ListItemSwitch` and this snapshot will become more readable
```
jest.mock("../../../app/components/ListItem", () => "ListItem");
jest.mock("../../../app/components/ListItemSwitch", () => "ListItemSwitch");
```